generate_parameter_library_py/setup.py

```diff
@@ -26,6 +26,14 @@
             "share/" + package_name + "/test",
             ["generate_parameter_library_py/test/invalid_parameter_type.yaml"],
         ),
+        (
+            "share/" + package_name + "/test",
+            ["generate_parameter_library_py/test/valid_parameters.yaml"],
```

Comment:
Thanks for adding this missing test!

Reply:
I also did it in a separate PR #100 but included this one here, until the other one is merged.